Modules/Loadable/Annotations/Widgets/qMRMLAnnotationROIWidget.cxx

```diff
@@ -100,11 +101,22 @@ vtkMRMLAnnotationROINode* qMRMLAnnotationROIWidget::mrmlROINode()const
 void qMRMLAnnotationROIWidget::setMRMLAnnotationROINode(vtkMRMLAnnotationROINode* roiNode)
 {
   Q_D(qMRMLAnnotationROIWidget);
-  qvtkReconnect(d->ROINode, roiNode, vtkCommand::ModifiedEvent,
+
+  if(!roiNode)
+    {
+    return;
+    }
```

Comment:
I don't think checking if `roiNode` is null is required. Removing the check will ensure the slots will be disconnected.